### PR TITLE
fix: unify github actions workflow using multiarch image of build-tools

### PR DESCRIPTION
**Pull request by xoanmm**
_Provide a description of what has been changed_

### Checklist

- [x] Commits are signed with Developer Certificate of Origin (DCO - [learn more](https://github.com/kedacore/keda/blob/main/CONTRIBUTING.md#developer-certificate-of-origin-signing-your-work))

Relates to #3092

### CODE REVIEW

File: .github/workflows/pr-validation.yml [hunk 1]
```diff
@@ -3,73 +3,17 @@ on:
...
-    env:
-      ARCH: arm64
+    name: validate - ${{ matrix.name }}
+    runs-on: ${{ matrix.runner }}
+    container: ghcr.io/kedacore/build-tools:main
+    strategy:
+       matrix:
+        include:
+        - runner: ARM64
+          name: amr64
```
Comment (JorTurFer) [inline .github/workflows/pr-validation.yml:13]: there is a typo here, arm64

File: .github/workflows/pr-validation.yml [hunk 2]
```diff
@@ -3,73 +3,17 @@ on:
...
+        - runner: ubuntu:latest
```
Comment (JorTurFer) [inline .github/workflows/pr-validation.yml:14]: I think the runner is `ubuntu-latest`

Comment (JorTurFer) [inline .github/workflows/pr-validation.yml:14]: I think the runner is `ubuntu-latest`

**Comment by xoanmm** [comment]
It's not possible use container with image `ghcr.io/kedacore/build-tools:main` in job `validate-arm64` from `main-build` workflow due it's using kind and the tests are running inside a container, [example of execution](https://github.com/kedacore/keda/runs/7355531170?check_suite_focus=true)

**Review by JorTurFer — APPROVED** [review]

**Pull request merged**